.github/workflows/test-dependencies.txt

```diff
@@ -0,0 +1,39 @@
+acl
```

Comment:
Optional for this PR, but - maybe cleanup https://github.com/openzfs/zfs/blob/master/.github/workflows/build-dependencies.txt from test deps and group them when needed?

Reply:
Yes, the build- and test- dependencies need to be re-checked.

Reply:
The splitting within `.github/workflows/scripts/split_functional_tests.sh` may also need some update... it's just a start. We will see how the runners differ in time... and can fine tune the things a bit.